bin/create_svg_manifest.js

```diff
@@ -0,0 +1,32 @@
+const glob = require('glob');
+const _ = require('lodash');
+const fs = require('fs'); // eslint-disable-line
+
+function reducer(acc, curr) {
+  const camel = curr.split('.')[0];
+
+  return Object.assign({}, acc, {
+    [_.camelCase(camel)]: `./${curr}`
```

Comment:
same comment with camel. i think this name is actually misleading. since you're camelizing it, it's probably not already camelized. is it perhaps the `fileName`?